<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Azael Avalos &lt;coproscefalo@gmail.com&gt;</author>
 <class>BootSettings</class>
 <widget class="QWidget" name="BootSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Boot Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="bootorderGroupBox">
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="bootOrderLayout">
      <item>
       <widget class="QTreeView" name="deviceList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonsLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="deferButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Defer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="preferButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Prefer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="wol_label">
        <property name="text">
         <string>Wake on LAN (WOL):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" alignment="Qt::AlignRight">
       <widget class="QCheckBox" name="wol_checkbox">
        <property name="toolTip">
         <string>Turns ON or awakens the laptop by a network message.</string>
        </property>
        <property name="whatsThis">
         <string>Turns ON or awakens the laptop by a network message.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="panel_power_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" alignment="Qt::AlignRight">
       <widget class="QCheckBox" name="panel_power_checkbox">
        <property name="toolTip">
         <string>Turns ON the laptop whenever you open the LID.</string>
        </property>
        <property name="whatsThis">
         <string>Turns ON the laptop whenever you open the LID.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="panel_power_label">
        <property name="text">
         <string>Panel Power ON:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="wok_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="wok_label">
        <property name="text">
         <string>Wake on Keyboard (WOK):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="wol_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" alignment="Qt::AlignRight">
       <widget class="QCheckBox" name="wok_checkbox">
        <property name="toolTip">
         <string>Awakens the laptop whenever you press the keyboard, when the computer is asleep.</string>
        </property>
        <property name="whatsThis">
         <string>Awakens the laptop whenever you press the keyboard, when the computer is asleep.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
